docs/packaging.md

```diff
@@ -11,27 +11,29 @@ Each zip has the following folder structure:
   - `/ServiceControl.Audit`
   - `/ServiceControl.Monitoring`
 
-## Assembly Mismatches
+## The mechanics
 
-There can be an issue when the main instance folder and the selected transport each have a copy of the same assembly. This can happen when the transport/persistence package and the instance each reference the same dependency, but select different versions. At install time, one version or the other will be copied into the instance binary folder and things may break unexpectedly at runtime.
+The `ServiceControlInstaller.Packaging` project is responsible to output to a `deploy` folder in the root all the required artifacts for `ServiceControl`, `ServiceControl.Audit`, and `ServiceControl.Monitoring`. Those artifacts are later needed by:
 
-To prevent this, the unit test `DeploymentPackageTests.DuplicatAssemblyFileSizesShouldMatch` tests the generated zip files for duplicated assemblies. If their file sizes match, then the test passes. If the file sizes do not match, then the test will fail with a message:
+- the `ServiceControlInstaller.Engine` project to create the above-mentioned required zip files
+- the `Particular.PlatformSample.ServiceControl` project to create the Platform sample required NuGet package
+- The Docker `dockerfile`(s) to build the Docker container images
 
-```
-  File sizes should match the ones in the ServiceControl.Audit folder. Check versions of dependencies.
-  Expected: <empty>
-  But was:  < "Transports/AzureStorageQueue/System.Runtime.CompilerServices.Unsafe.dll", "Transports/AzureStorageQueue/System.Threading.Tasks.Extensions.dll" >
+## Assembly version mismatches
 
-   at Tests.DeploymentPackageTests.DuplicateAssemblyFileSizesShouldMatch() in /_/src/ServiceControlInstaller.Packaging.UnitTests/DeploymentPackageTests.cs:line 19
-```
+There can be an issue when the main instance folder and the selected transport each have a copy of the same assembly. This can happen when the transport/persistence package and the instance each reference the same dependency, but select different versions. At install time, one version or the other will be copied into the instance binary folder and things may break unexpectedly at runtime.
 
-> File sizes should match the ones in the **ServiceControl.Audit** folder. Check versions of dependencies.
+To prevent this, the unit test `DeploymentPackageTests.DuplicateAssemblyShouldHaveMatchingVersions` tests the generated zip files for duplicated assemblies. If their versions match, then the test passes. If not then the test will fail with:
```

Comment:
```suggestion
To prevent this, the unit test `DeploymentPackageTests.DuplicateAssemblyShouldHaveMatchingVersions` tests if duplicated assemblies might be deployed. If their versions match, then the test passes. If not then the test will fail with:
```